src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -518,7 +526,8 @@
                 color: $clr-dark-gray;
                 border: none;
                 background: none;
-                padding: 0;
+                padding: 8px;
```

Comment:
Same for the ones here

Reply:
If not a variable, this should probably be using our existing baselineRem variables.

Reply:
Settled on 
``` 
            .datagrid-action-toggle {
                padding: $clr_baselineRem_0_375 - $clr-table-borderwidth;
                margin: ($clr_baselineRem_0_125 - 1) $horizontalPadding;
            }```